HackYourFuture-CPH/fp-class12
# Fix: image name

# Description

I renamed a file from close-outline 1.png to close-outline1.png.

Fixes # (issue)

# How to test?

Go to src/client/assets/images and find close-outlline1.png. Make sure that there's no space in the file name. 

# Checklist

- [x] I have merged the latest version of the source branch (typically develop) into this branch before pushing my changes
